website/package.json

```diff
@@ -19,6 +19,7 @@
     "@docusaurus/preset-classic": "3.5.2",
     "@mdx-js/react": "^3.0.0",
     "clsx": "^2.0.0",
+    "docusaurus": "^1.14.7",
```

Comment:
You don't need this dependency. I can remove it and push to this PR if you'd like. lmk.